core/sds-aspect-model-java-generator/src/main/resources/java-enumeration.vm

```diff
@@ -12,7 +12,9 @@
  *#
 #parse( "java-enumeration-class-body-lib.vm" )
 #set( $classBody = $util.applyImports( "#javaEnumerationClassBody()", $codeGenerationConfig ) )
+#if( $codeGenerationConfig.doExecuteLibraryMacros() )
 #copyrightHeader
```

Comment:
Would it make sense to change the name of the macro to something more generic, e.g. "fileHeader"?